tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_laps.py

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.clock.app import Clock
+
+class TestClockRunStopWatch(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+
+        self.clock = Clock(self.marionette)
+        self.clock.launch()
+
+    def test_click_run_stopwatch_laps(self):
+
+        stopwatch_view = self.clock.switch_view("stopwatch")
+
+        self.assertEqual(stopwatch_view.current_time, '00:00.00')
+
+        stopwatch_view.tap_start()
+
```

Comment:
I would add a small import time time.sleep(0.1) here. That makes the following assert pass on b2g desktop then.
On the Flame device this wasn't an issue, apparently, because everything is slower there.